babel.config.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const isDev = process.env.npm_lifecycle_event === 'dev';
+
+module.exports = function(api) {
+  const presets = [
+    [
+      '@babel/preset-env',
+      Object.assign(
+        {},
+        isDev ? { targets: { node: 'current' } } : { targets: { node: 6 } }
```

Comment:
Since we do not officially support Node versions below 6, I've explicitly locked transpilation at that version. This results in much more readable (and debuggable) code being generated.